e2e/cypress/integration/plugins/plugin_buttons_spec.js

```diff
@@ -50,15 +46,13 @@ describe('collapse on 5 plugin buttons', () => {
         cy.visit(`/${testTeam.name}/channels/town-square`);
 
         // # Upload and enable test plugin with 5 channel header icons
-        cy.apiUploadPlugin('com.mattermost.test-plugin-0.1.0.tar.gz').then(() => {
-            cy.apiEnablePluginById('com.mattermost.test-plugin');
+        cy.apiUploadAndEnablePlugin(testPlugin).then(() => {
```

Comment:
This spec is failing due to outdated "testPlugin" which will be addressed separately by Integrations team - https://mattermost.atlassian.net/browse/MM-35110